### DISCUSSION TITLE
热门视频消失不见了？

### DISCUSSION THREAD

**Discussion by Nazrin720** [the1812/Bilibili-Evolved#1427] [综合讨论 / General]
跟图片里面的一样版本是1.11.5
![QQ截图20210106225123](https://user-images.githubusercontent.com/63951577/103782194-1ee8d000-5072-11eb-95bc-02291f41bbdb.png)

**Comment by Nazrin720** [comment]
https://imgchr.com/i/sZexfI
图片的链接

**Comment by Geeyun-JY3** [comment]
提之前能不能先看看第一页 issue 或搜一下：#1415。~就在第一页~还就是置顶。
> 查看置顶问题
在 Pinned issues 中会包含一些近期常见问题, 已有的其他问题也会在下面列出, 请务必查看避免重复发起

https://github.com/the1812/Bilibili-Evolved/blob/preview/issue-rules.md
> 提示: 最好先在预览离线版(Preview Offline)中测试一下, 有些问题可能已经修复了但还未正式发布.

而且最新预览离线版已经修了。
